lib/trie/node.ex

```diff
@@ -91,6 +91,7 @@ defmodule MerklePatriciaTree.Trie.Node do
     case Storage.get_node(trie) do
       nil -> :empty
       <<>> -> :empty
+      :not_found -> :empty
```

Comment:
I think this makes sense.. when you try to walk down a tree with non-existent leaves, you just get empty trees?

Reply:
Yeah, I guess. If we find out in our implementations that we need to know which type of emptiness we got than I guess we can come back and add them.